modules/kwin.nix

```diff
@@ -307,5 +314,12 @@ in
           (virtualDesktopNameAttrs cfg.kwin.virtualDesktops.names)
         ];
       })
+
+      # Borderless maximized windows
+      (mkIf (cfg.kwin.borderlessMaximizedWindows != null) {
+        Windows = {
+          BorderlessMaximizedWindows = cfg.kwin.BorderlessMaximizedWindows;
```

Comment:
Should be `cfg.kwin.borderlessMaximizedWindows` (small B).

Reply:
You're right.